.github/workflows/tests.yaml

```diff
@@ -50,6 +50,37 @@ jobs:
       run: |
         coveralls
 
+  # A job that runs the test suite using the development version of the
+  # deepcell dependencies: deepcell-tracking and deepcell-toolbox.
+  # This ensures consistency across unreleased versions of deepcell libraries
+  test-dev:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8", "3.9"]
```

Comment:
Should python 3.10 be added to the list or is that tied to a different PR?

Reply:
Nice catch - yeah it should be added, but not yet. I wanted to get this job in place so we could test against `deepcell-tracking` once 3.10 support is added there by vanvalenlab/deepcell-tracking#111. Updating `deepcell-tf` to add 3.10 support is last on the list (PR incoming soon).